x-pack/solutions/security/plugins/cloud_security_posture/server/plugin.ts

```diff
@@ -151,9 +155,17 @@ export class CspPlugin
           'packagePolicyUpdate',
           async (
             packagePolicy: UpdatePackagePolicy,
-            soClient: SavedObjectsClientContract
+            soClient: SavedObjectsClientContract,
+            esClient: ElasticsearchClient
           ): Promise<UpdatePackagePolicy> => {
             if (isCspPackage(packagePolicy.package?.name)) {
+              const isIntegrationVersionIncludesTransformAsset =
```

Comment:
let's maybe move it to a small utility? we do this "if version > 2 then includes transform" check multiple times